recipes/ignition-math/all/conanfile.py

```diff
@@ -104,46 +105,63 @@ def package(self):
+        """.format(major=version.major, minor=version.minor, patch=version.patch))
+        tools.save(module_file, content)
+
+
     def package_info(self):
         version_major = tools.Version(self.version).major
-        self.cpp_info.names["cmake_find_package"] = "ignition-math{}".format(version_major)
-        self.cpp_info.names["cmake_find_package_multi"] = "ignition-math{}".format(version_major)
-
-        # cmake_find_package filename: ignition-math6-config.cmake
-        self.cpp_info.components["libignition-math"].libs = ["ignition-math{}".format(version_major)]
-        self.cpp_info.components["libignition-math"].includedirs.append("include/ignition/math{}".format(version_major))
-        self.cpp_info.components["libignition-math"].names["cmake_find_package"] = "ignition-math{}".format(version_major)
-        self.cpp_info.components["libignition-math"].names["cmake_find_package_multi"] = "ignition-math{}".format(version_major)
-        self.cpp_info.components["libignition-math"].names["pkg_config"] = "ignition-math{}".format(version_major)
-
-        # FIXME: create in file ignition-math6-eigen3-config.cmake
-        self.cpp_info.components["libignition-math-eigen3"].libs = []
-        self.cpp_info.components["libignition-math-eigen3"].requires = ["libignition-math", "eigen::eigen"]
-        self.cpp_info.components["libignition-math-eigen3"].names["cmake_find_package"] = "ignition-math{}-eigen3".format(version_major)
-        self.cpp_info.components["libignition-math-eigen3"].names["cmake_find_package_multi"] = "ignition-math{}-eigen3".format(version_major)
-        self.cpp_info.components["libignition-math-eigen3"].names["pkg_config"] = "ignition-math{}-eigen3".format(version_major)
-
-        # FIXME: create in file ignition-math6-all-config.cmake
-        self.cpp_info.components["libignition-math-all"].libs = []
-        self.cpp_info.components["libignition-math-all"].requires = ["libignition-math-eigen3"]
-        self.cpp_info.components["libignition-math-all"].names["cmake_find_package"] = "ignition-math{}-all".format(version_major)
-        self.cpp_info.components["libignition-math-all"].names["cmake_find_package_multi"] = "ignition-math{}-all".format(version_major)
-        self.cpp_info.components["libignition-math-all"].requires.append("eigen::eigen")
-        self.cpp_info.components["libignition-math-all"].requires.append("doxygen::doxygen")
-        self.cpp_info.components["libignition-math-all"].requires.append("swig::swig")
-
-        self.env_info.LD_LIBRARY_PATH.extend([
-            os.path.join(self.package_folder, x) for x in self.cpp_info.libdirs
-        ])
-        self.env_info.PATH.extend([
-            os.path.join(self.package_folder, x) for x in self.cpp_info.bindirs
-        ])
+        lib_name = f"ignition-math{version_major}"
+        build_dirs = os.path.join("lib", "cmake")
```

Comment:
In my comment about hard coding the path, I meant to return `os.path.join("lib", "cmake")` in a method e.g. called `_module_dir_rel_path`. And let `_module_file_rel_path` use that method.

Reply:
@madebr . Now does my change reflect the intent ?

Reply:
Excellent! Thanks!